packages/plugin-dts/src/dts.ts

```diff
@@ -24,7 +24,8 @@ export async function generateDts(data: DtsGenOptions): Promise<void> {
     throw new Error();
   }
   const { options: rawCompilerOptions } = loadTsconfig(configPath);
-  const rootDir = rawCompilerOptions.rootDir ?? 'src';
+  const rootDir =
+    rawCompilerOptions.rootDir ?? join(dirname(configPath), 'src');
```

Comment:
Why do we need to add to provide a fallback to `src`, although is commonly used. Is it better to require user to define it explicitly?

Reply:
We need rootDir to calculate dts entry file in bundle mode, if user do not config, fallback will help.

Reply:
In 95% cases, `src` helps. What about in other `5%`.